Backpack-SwiftUI/Calendar/Classes/CalendarMonthHeader.swift

```diff
@@ -66,7 +66,9 @@ struct CalendarMonthHeader: View {
 
         let lowerBound = max(validRange.lowerBound, monthRange.lowerBound)
         let upperBound = min(validRange.upperBound, monthRange.upperBound)
-
+        if lowerBound > upperBound {
```

Comment:
Reviewer note: In cases where a consumer incorrectly set a whole month button, this condition would protect from a crash as with the new changes it's possible theoretically to set a whole month button to the month outside of a valid range